src/main/webapp/partials/instructorHelpSearch.jsp

```diff
@@ -6,116 +6,120 @@
+                        </th>
+                        <th class="button-sort-none" onclick="toggleSort(this)">
+                          Team <span class="icon-sort unsorted"></span>
+                        </th>
+                        <th class="button-sort-none" onclick="toggleSort(this)">
+                          Student Name <span class="icon-sort unsorted"></span>
+                        </th>
+                        <th class="button-sort-none" onclick="toggleSort(this)">
+                          Status <span class="icon-sort unsorted"></span>
+                        </th>
+                        <th class="button-sort-none" onclick="toggleSort(this)">
+                          Email <span class="icon-sort unsorted"></span>
+                        </th>
+                        <th>Action(s)</th>
+                      </tr>
+                    </thead>
+                    <tbody>
+                      <tr class="student_row">
+                        <td>
+                          <div class="profile-pic-icon-click align-center">
+                            <a class="student-profile-pic-view-link btn-link">View Photo</a>
+                            <img src="" alt="No Image Given" class="hidden">
+                          </div>
+                        </td>
+                        <td>Section A</td>
+                        <td>Team A</td>
+                        <td><span class="highlight">Alice</span> Betsy</td>
+                        <td class="align-center">Joined</td>
+                        <td><span class="highlight">alice</span>@email.com</td>
+                        <td class="no-print align-center">
+                          <a class="btn btn-default btn-xs" title="View details of the student"
+                              href="" target="_blank" rel="noopener noreferrer" data-toggle="tooltip" data-placement="top">View</a>
+                          <a class="btn btn-default btn-xs" title="Use this to edit the details of this student. <br>To edit multiple students in one go, you can use the enroll page: <br>Simply enroll students using the updated data and existing data will be updated accordingly"
+                              href="" target="_blank" rel="noopener noreferrer" data-toggle="tooltip" data-placement="top">Edit</a>
+                          <a class="course-student-delete-link btn btn-default btn-xs"
+                              title="Delete the student and the corresponding submissions from the course" href="" data-toggle="tooltip" data-placement="top">Delete</a>
+                          <a class="btn btn-default btn-xs" href="" title="View all data about this student" target="_blank" rel="noopener noreferrer"
+                              data-toggle="tooltip" data-placement="top">All Records</a>
+                        </td>
+                      </tr>
+                    </tbody>
+                  </table>
+                </div>
               </div>
```

Comment:
You forget to add example frame to this table
![screen shot 2018-01-10 at 8 19 33 pm](https://user-images.githubusercontent.com/13977235/34772647-b2b76cba-f643-11e7-9198-8b5c558cc743.png)